resources/local/chainspec.toml.in

```diff
@@ -118,13 +118,13 @@ max_ttl = '18 hours'
 # Maximum transaction size in bytes.  Size is of the transaction when serialized via ToBytes.
 max_transaction_size = 1_048_576
 # Maximum number of transfer transactions allowed in a block.
-block_max_transfer_count = 1000
```

Comment:
Where do the changes to the chainspec come from?

Reply:
I believe it was a discussion either in a test huddle or on the testing slack channel.  These values should have been changed a while back apparently.